src/tests.js

```diff
@@ -64,11 +54,18 @@ function Test(component, config) {
     }
   }
 
+  const mixins = config.mixins === undefined ? {} : config.mixins
+
   return testComponent.mixin({
     find: Find,
     setState: SetState,
-    simulate: Simulate
+    simulate: Simulate,
+    ...mixins
   })
 }
 
+export const addMixins = (Test, mixins) => {
+  return (component, config) => Test(component, { ...config, mixins })
+}
```

Comment:
I actually considered that. But decided not to do it because of its implicit nature. I don't want to make a function impure unless it is necessary. I like being explicit. When I read `CustomTest = addMixins(mixins)`, I find asking myself addMixins to what. That's my reasoning. What do you think??